callosum/peer.py

```diff
@@ -1,22 +1,43 @@
 import asyncio
 import functools
 import logging
-from typing import Callable, Type
+from typing import (
+    Callable, Type,
+    Mapping, List,
+    Any,
+)
+from collections import defaultdict
+from datetime import datetime
+from dateutil.tz import tzutc
 import secrets
 
 import aiojobs
+from aiohttp import web
+from aiojobs.aiohttp import get_scheduler_from_app
 from async_timeout import timeout
 import attr
 
 from .auth import AbstractAuthenticator
 from .compat import current_loop
 from .exceptions import ServerError, HandlerError
+from .eventmessage import (
+    EventMessage,
+    EventTypes,
+    EventHandler,
+)
 from .message import RPCMessage, RPCMessageTypes
 from .ordering import (
     AsyncResolver, AbstractAsyncScheduler,
     KeySerializedAsyncScheduler,
 )
-from .lower import AbstractAddress, BaseTransport
+from .lower import (
+    AbstractAddress,
+    BaseTransport,
+)
+from .lower.redis import (
+    TransportType,
```

Comment:
I think this is a leak of abstraction, as lower-specific detail is exposed in the higher abstraction module.
Could we avoid this?

Reply:
Now there is no this import statement and neither among Publisher, Subscriber, Peer classes uses it in any possible way.